source/languages/en/riak/dev/taste-of-riak/querying-golang.md

```diff
@@ -0,0 +1,562 @@
+---
+title: "Taste of Riak: Querying with Go"
+project: riak
+version: 1.3.1+
```

Comment:
I know, I'm a broken record :) But I want to check the version compatibility on this one, too.